cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -177,6 +180,15 @@ type APIServer struct {
 type DNS struct {
 	// ImageMeta allows to customize the image used for the DNS component
```

Comment:
```suggestion
	// ImageMeta allows to customize the image used for the DNS addon
```

Reply:
this is an existing comment, but please change it.

Reply:
Done.